docs/source/commandline.md

```diff
@@ -91,6 +91,12 @@ You can pass an absolute or relative path.
 
 **Flags for default packages**
 
+`--prototype`
```

Comment:
Maybe mention what the prototype packages are? Or are you going to add them to the Packages table?

Reply:
> Maybe mention what the prototype packages are

I've updated the text to the following:
```txt
Creates a package with the prototype purpose packages(`autopublish` and `insecure`) 
if you use them you can change your collections quickly, 
but it is not supposed to be used in production.
For more information about security you can check
it [here](https://guide.meteor.com/security.html#checklist)
It can be used together with other flags that create apps such as `--react` or `--typescript`.
```